src/NBitcoin/Network.cs

```diff
@@ -186,7 +186,7 @@ public class PrimaryAuthenticators
 
         public uint Quorum { get; private set; }
 
-        public PrimaryAuthenticators(Network network, string[] base58Addresses, uint quorum)
+        public PrimaryAuthenticators(string[] base58Addresses, uint quorum)
```

Comment:
This is removing an unused argument and can be moved to a separate PR to improve the reviewability of this PR.